app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name, age):
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    result_list = []
+    for person in people:
+        item = Person(person["name"], person["age"])
```

Comment:
```suggestion
        person = Person(person["name"], person["age"])
```

Reply:
`item` isn't a descriptive name as well
I suggest you use the `person` variable to store a `Person` instance and `person_dict` to store a dictionary with information about the person you are going to create